docfx_yaml/extension.py

```diff
@@ -134,6 +134,21 @@ class Bcolors:
     DEPRECATED: 'deprecated',
 }
 
+_SUMMARY_TYPE_BY_ITEM_TYPE = {
```

Comment:
Nit: Maybe use `_SUMMARY_GROUP` instead? The phrasing can get a little bit confusing because you're working with Python types as well.

Reply:
This was a preferred style for mappings (summary_by_item[item] gives summary) - I could also just omit the types if that makes it better